packages/retail-ui-extensions/src/extension-api/types/modal.ts

```diff
@@ -0,0 +1,10 @@
+export interface Modal {
+  /**
+   * Path should be used to determine which Screen or Navigator to return for the Modal extension point.
+   */
+  path: string;
+  /**
+   * Params can be used when presenting a modal to pass any initial parameters or attach callbacks.
+   */
+  params?: any;
```

Comment:
I think at some point someone asked me this question before. 

`any` makes the most sense because the best other thing we could use otherwise would be a dictionary, I suppose, but it would end up being `string` to `any` type anyway, so using a straight `any` seems like the best option for now. And using `routes` for strict typing would be overly complicated at this point.